### DISCUSSION TITLE
3d printed enclosure

### DISCUSSION THREAD

**Discussion by liselotte111** [CrashOverride85/zc95#84] [General]
I forgot to order the enclosure at Mouser and since the cad was available I decided to edit it. The enclosure can be printed with eg PETG or ABS. You'll need 5mm heatset inserts. I've attached the .stl and .step files

![photo_2024-01-05_22-49-08](https://github.com/CrashOverride85/zc95/assets/36988110/6ee9938e-389e-49c9-aaea-52e264011f7c)

[ZC95 case.zip](https://github.com/CrashOverride85/zc95/files/13917569/ZC95.case.zip)

**Comment by CrashOverride85** [comment]
Very nice  🙂, didn't realise there was a cad file for the case

**Comment by moarkaboom** [comment]
This is great! Thanks for the STEP!

Edit: Just finished printing this and setting the heat inserts. Its really great! One minor thing I was wondering, you only used two screws to keep the case closed, why is that? I would prefer four screws and would remix your file but am unsure if there is enough room for four screw posts with the PCBs inside. Can you make an assessment?

**Comment by moarkaboom** [comment]
I would like to customize my case a bit, is there a way to import the PCBs into CAD/Fusion?

**Comment by CrashOverride85** [comment]
What sort of inserts did you use? M5 seems rather too large?
